tsdb/engine/tsm1/file_store.go

```diff
@@ -121,6 +124,13 @@ func (f *FileStore) Count() int {
 	return len(f.files)
 }
 
+// Files returns TSM files currently loaded.
+func (f *FileStore) Files() []TSMFile {
```

Comment:
I assumed so, but other people/code might see it as exported and think it's safe to access.